packages/core/src/animation/Animator.ts

```diff
@@ -275,17 +274,30 @@ export class Animator extends Component {
     const { curveOwnerPool: layerCurveOwnerPool } = animatorLayerData;
     const { _curveBindings: curves } = animatorState.clip;
 
+    this._needRevertCurveOwners.length = 0;
```

Comment:
This maybe have bug, what you do is per single Animator state, not consider global